test/integration/auth.spec.ts

```diff
@@ -91,6 +91,10 @@ function randomOidcProviderId(): string {
   return 'oidc.' + generateRandomString(10, false).toLowerCase();
 }
 
+function client_auth(): FirebaseAuth {
```

Comment:
Please use camel case `clientAuth` for function names here and elsewhere.

Reply:
Done.